### PR TITLE
modal added

### PR DESCRIPTION
**Pull request by bishnoimukesh**
# Change Logs

## Snapshot:
modal

## Deploy Preview Link: https://deploy-preview-5--comp-ui.netlify.app/components/modal/modal

### CODE REVIEW

**Comment by amirnathani9** [comment]
Looks Good. Make the suggested changes.

File: components/modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
+:root{
+    --modal-background-color: #000;
+    --modal-text-color: #fff;
+    --model-content-background-color: #2a9d8f;
+}
```
Comment (devdeadviz) [inline components/modal/modal.css:5]: You can create a variable stylesheet and put them inside it.

File: components/modal/modal.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    background-color: var(--model-content-background-color);
+    margin: auto;
+    padding: 3em;
+    border: 1px solid var(--modal-text-color);
+    width: 70%;
+    color: var(--modal-text-color);
+}  
+
+.close {
+    color: #fff;
```
Comment (devdeadviz) [inline components/modal/modal.css]: You can also create a variable for this white color.

**Review by devdeadviz — APPROVED** [review]
Added some suggestions.

File: components/modal/modal.css (hunk 1 above)
Comment (bishnoimukesh) [reply components/modal/modal.css:5]: will take care of it

**Pull request merged**